ports/glutin/window.rs

```diff
@@ -345,16 +345,16 @@ impl Window {
                     self.pending_key_event_char.set(Some(ch));
                 }
             }
-            Event::KeyboardInput(element_state, scan_code, Some(virtual_key_code)) => {
```

Comment:
I'm actually more of the opinion that we should split this up into different match arms. The common code between the two match arms should just be refactored into a separate function, since this arm is already pretty big as it is. What do you think?